src/App.js

```diff
@@ -107,11 +107,9 @@ function App() {
     <Router>
       <div>
         <nav>
-          <ul>
-            <li>
-              <Link to="/">Sélectionner une room</Link>
-            </li>
-          </ul>
+          <Button style={{ margin: "14px" }}>
+            <Link to="/">Menu de sélection de compteur</Link>
```

Comment:
Je crois meme que le Button peut etre rendu comme link avec un as et donc choper les props de la class link
genre <Button style(...) as="Link" to="/">menu</Button>

Reply:
En effet, c'est plutôt stylé ! Il va vraiment falloir que j'aille fouiller les options de react, il y a beaucoup de trucs sympas !